src/components/design-system/atoms/burger-icon/burger-icon.tsx

```diff
@@ -0,0 +1,24 @@
+import { Disclosure } from '@headlessui/react';
+import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline';
+
+export default function BurgerIcon({
+  isOpen,
+  accessibilityName,
+}: {
+  isOpen: boolean;
+  accessibilityName: string;
+}) {
+  return (
+    <div className="-mr-2 flex items-center sm:hidden">
+      <Disclosure.Button className="relative inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-100 hover:text-gray-500 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-indigo-500">
```

Comment:
Enfaite la main color c'est pirate gold, du coup tu peux remplacer tout les indigo en pirategold ?

```suggestion
      <Disclosure.Button className="relative inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-100 hover:text-gray-500 focus:outline-none focus:ring-2 focus:ring-inset focus:ring-pirateGold">
```

Reply:
done